spec/integrations/shared/base_render_examples.rb

```diff
@@ -425,4 +425,22 @@ def self.unless_method(_field_name, _object, _options)
     end
     it('returns json with values derived from options') { should eq(result) }
   end
+
+  context 'Given blueprint has a transformer' do
+    subject { blueprint.render(obj) }
+    let(:result) { '{"id":' + obj_id + ',"full_name":"Meg Ryan"}' }
+    let(:blueprint) do
+      DynamicFieldsTransformer = Class.new(Blueprinter::Transformer) do
+        def transform(result_hash, object, options={})
+          dynamic_fields = (object.is_a? Hash) ? object[:dynamic_fields] : object.dynamic_fields
+          result_hash.merge!(dynamic_fields)
+        end
+      end
+      Class.new(Blueprinter::Base) do
+        identifier :id
+        transform DynamicFieldsTransformer
+      end
+    end
+    it('returns json with values derived from options') { should eq(result) }
+  end
```

Comment:
Would you also add a test to also ensure that the transformers are also inherited? There's some inheritance specs here that you can add to: https://github.com/procore/blueprinter/blame/4ba12dd14edf340f9a4a95f1668785adc8c21ca6/spec/integrations/base_spec.rb#L326-L348

Reply:
Test case has been added.